templates/store/snap-details.html

```diff
@@ -192,22 +192,48 @@ <h4>Share this snap</h4>
   </div>
 
   {% if not IS_BRAND_STORE %}
-    <div data-js="blog-articles" style="display: none;">
+    {% if package_name == "hiri" %}
```

Comment:
Not sure how long it is supposed to live here, but maybe we could move it to some partial, so it doesn't clutter main template too much?